tooling/v8-snapshot/src/setup/consolidate-deps.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path'
+import fs from 'fs-extra'
+import { sync as glob } from 'glob'
```

Comment:
why not import glob and just use glob.sync(). That seems more readable.

Reply:
Updated: [cypress-io/cypress@`ec34a53` (#23732)](https://github.com/cypress-io/cypress/pull/23732/commits/ec34a537a4879edcfbc0409b0f81f16a94e97ee9)